### DISCUSSION TITLE
Can a user have multiple emails?

### DISCUSSION THREAD

**Discussion by Falci** [supabase/supabase#20611] [Questions]
Is it possible to add multiple identity with provider `email` ?

For my use case, after sign in, the user can add their other emails, which should send the validation email to confirm. Instead of creating a new account that would be added to the same user

**Comment by GaryAustin1** [comment]
No (pretty sure). Certainly not before this was recently added for linking identities: https://supabase.com/docs/guides/auth/auth-identity-linking
You can read thru it, but I believe it only works to link OAuth providers to a user with a single email address.
